### DISCUSSION TITLE
Live tracking on iPhone - disable sleep

### DISCUSSION THREAD

**Discussion by Ernie3128** [jaybo/deepzoom-blog#27] [General]
I would like to use DeepZoom as a live navigating source on my iPhone instead of other mapping apps (maps, Navionics, etc) but i am running into issues with device sleep mode. Are there settings within DeepZoom that can prevent device going to sleep? I suspect no since this is a website rather than an app.  Any advice would be helpful. TIA

**Comment by jaybo** [comment]
As long as you keep DeepZoom in the foreground and prevent the display from locking (Settings.Display&Brightness.Auto-Lock.Never), you can continuously track your location.  Definitely a hassle, and this option isn't battery efficient.  This action must be manually initiated by the user; DeepZoom can't do it on your behalf.

The better solution is for the WC3 to advocate completion of the Geolocation-Sensor API (https://w3c.github.io/geolocation-sensor/) which would enable background location collection by web apps.  Unfortunately, Apple & Google et al. aren't keen to make the necessary browser changes which would make web apps as performant as native apps, (possibly because this would endanger their 30% sales cut?), so progress on this has been glacial.
